TUTORIAL.md

```diff
@@ -390,23 +390,23 @@ Proceed or Abort
 If you click **Proceed**, the build will proceed as before.
 First, go to the Jenkins main page and look at the **Build Executor Status** widget.
 
-* You will see an unnumbered entry under **master** named  **flowname #10**; executors #1 and #2 on the master are idle.
-* You will also see an entry under your slave, in a numbered row (probably #1) called **Building part of flowname #10**.
+* You will see an unnumbered entry under **master** named  **jobname #10**; executors #1 and #2 on the master are idle.
+* You will also see an entry under your slave, in a numbered row (probably #1) called **Building part of jobname #10**.
 
-Why are there two executors consumed by one flow build?
+Why are there two executors consumed by one Pipeline build?
 
-* Every flow build itself runs on the master, using a **flyweight executor** — an uncounted slot that is assumed to not take any significant computational power.
+* Every Pipeline build itself runs on the master, using a **flyweight executor** — an uncounted slot that is assumed to not take any significant computational power.
```

Comment:
I think we are fine with the term “build” in this context; the question of whether we should introduce a new generic term to correspond to `WorkflowRun` was never really settled.